google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -682,6 +686,77 @@ def copy destination_table, create: nil, write: nil, dryrun: nil,
         end
 
         ##
+        # Copies the data from the table to another table using a synchronous
+        # method that blocks for a response. In this method, a {CopyJob} is
```

Comment:
I wouldn't mention CopyJob here. I would just say the method blocks until the copy operation is complete, or an error is raised.

Reply:
Ok, I'll update for these.